<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Miscellaneous_View</class>
 <widget class="QMainWindow" name="Miscellaneous_View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Miscellaneous</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="conditions_groupBox">
      <property name="title">
       <string>Conditions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Buff</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="buffName_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>utito tempo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="buffMana_lineEdit">
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhDigitsOnly|Qt::InputMethodHint::ImhLowercaseOnly|Qt::InputMethodHint::ImhPreferLatin</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>290</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="buff_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Anty Paralyse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="paralName_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>exura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="paralMana_lineEdit">
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhDigitsOnly</set>
           </property>
           <property name="placeholderText">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="paral_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Mana Shield</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="shieldName_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>utamo vita</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="shieldMana_lineEdit">
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhDigitsOnly|Qt::InputMethodHint::ImhPreferNumbers</set>
           </property>
           <property name="placeholderText">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="shield_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Haste</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hasteName_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>utani hur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hasteMana_lineEdit">
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhDigitsOnly|Qt::InputMethodHint::ImhFormattedNumbersOnly|Qt::InputMethodHint::ImhLowercaseOnly</set>
           </property>
           <property name="placeholderText">
            <string>60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="haste_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="others_groupBox">
      <property name="title">
       <string>Others</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Refil Ammo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ammoID_lineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Ammo ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="refilAmmo_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Auto Ring</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ringID_lineEdit">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Ring ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ringOption_comboBox">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>HP%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MP%</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ringValue_lineEdit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>40</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoRing_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Auto Amulet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="amuletID_lineEdit">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>0</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Amulet ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="amuletOption_comboBox">
           <property name="minimumSize">
            <size>
             <width>65</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>HP%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MP%</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="amuletValue_lineEdit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>40</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoAmulet_checkBox">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
